test/unittests/execute_floating_point_test.cpp

```diff
@@ -264,6 +264,8 @@ TYPED_TEST(execute_floating_point_types, binop_nan_propagation)
     constexpr Instr opcodes[] = {
         Instr::f32_add,
         Instr::f32_div,
+        Instr::f32_min,
+        Instr::f32_max,
```

Comment:
The spec says nothing about canonical nans, yet all these tests below pass?

Reply:
It does. Implicitly, by using `nans_N{}` pseudo-function. I.e. if all NaN inputs are canonical NaNs the output must be a canonical NaN as well. Otherwise, an arithmetic NaN is fine.